jobs/config.json

```diff
@@ -337,6 +337,27 @@
       "sig-cli"
     ]
   },
+  "ci-kubernetes-e2e-gce-1-6-1-7-master-skew": {
+    "args": [
+      "--check-version-skew=false",
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest-1.6",
+      "--extract=ci/latest-1.7",
+      "--gcp-zone=us-central1-f",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-kubelet-ci-version=latest-1.6",
+      "--kubernetes-anywhere-kubernetes-version=latest-1.7",
```

Comment:
If I recall correctly, `latest-1.7` will resolve to the latest release of the control plane, which will generally be point releases now that 1.7.0 has gone out, so this will only test `1.7.6`, then `1.7.7`, then `1.7.8`, etc., after they've already been released. I thought there was a different string to actually test the latest CI build of the control plane, so we can catch bugs before the next point release.

I don't think we actually want `ci/latest` (which would be the absolute latest `master` build), but do we have `ci-1.7/latest` or something similar to get the latest CI build for the release branch?

Reply:
Actually, it might be that the code to support using CI control plane builds wasn't actually merged until 1.8, in which case, maybe this is the best we can do for older skew tests.

Reply:
I checked and kubeadm does not recognizes ci/latest-1.7 or ci-1.7/latest for --kubernetes-version flag. So I am leaving it as it is.

Reply:
Sounds good to me. Sorry for the red herring!

Reply:
FWIW, `ci/latest-1.8`, _should_ be supported in v1.7-v1.8 skew testing, once we have one test-infra bug cleared out. The feature was added in v1.8 kubeadm